drivers/power_domain/Kconfig

```diff
@@ -13,4 +13,9 @@ config POWER_DOMAIN_GPIO
 	depends on GPIO
 	depends on TIMEOUT_64BIT
 
+config POWER_DOMAIN_INTEL_ADSP
+	bool "Use intel power gating mechanisms"
```

Comment:
Intel capitalized? Maybe mention also the audio DSP Part in the description (not a generic Intel power gating mechanism).

Reply:
Is it correct that this doesn't depend on any other config option?

Reply:
Capitalized and added ADSP part. I don't think it should depend on anything else.